src/panels/TcpDumpPanel.ts

```diff
@@ -53,6 +60,20 @@ function getCaptureFromFilePath(filePath: string): string | null {
     return fileMatch && fileMatch[1];
 }
 
+// Test function to verify escaping
+function testEscapeRegExp() {
```

Comment:
💡 Idea: Can we please move this to the util file for latter wider consumption? if you think its a good idea? we can then test it via test suite run.

Reply:
Added a mocha test suite for the escaping function